.gitignore

```diff
@@ -15,4 +15,6 @@ working*
 
 App.config.mine
 src/packages/*
-!src/packages/repositories.config
\ No newline at end of file
+!src/packages/repositories.config
```

Comment:
I'm not sure why I had -! to begin with (but it worked) and why you changed it to +!, but I'm pretty sure !/src/packages/repositories.config does the job.